Makefile

```diff
@@ -143,3 +143,6 @@ runtime:
 
 runtime-ci:
 	cargo b --profile ci --all-features -p cairo-native-runtime && cp target/ci/libcairo_native_runtime.a .
+
+fuzz: check-llvm needs-cairo2 build-alexandria runtime-ci
+	cargo +nightly fuzz run fuzz-sierra
```

Comment:
Hi @hhamud !
Is it necessary to use +nightly?
Can we avoid it for the moment?

Reply:
its required unfortunately.

Reply:
@pefontana is `+nightly` still an absolute blocker? cargo fuzzer requires it though I am not sure if honggfuzz does.